### DISCUSSION TITLE
2FA - Unacceptable , Brazil (+55) not on the list

### DISCUSSION THREAD

**Discussion by davidAlexander08** [community/community#74824] [Other Feature Feedback, Questions, & Ideas]
![2FA](https://github.com/community/community/assets/142263303/f98056d1-a088-4cdb-884d-56b6f96e5d66)

It is unacceptable that Brazil isn't listed as one of the countries for 2FA authentication.

**Comment by kheitmannf** [comment]
Yes, it is unacceptable. Chile neither appears in the list. It is outrageous, the software company that sent software to be buried in the arctic vault for future generations does not have Brazil and Chile in their SMS list.

I bet they copied the list many many years ago and then didn't bother to update it. And they don't care to update it now, otherwise they will have replied to you. TOTP is full of bugs and errors, I tested my recovery codes and sometimes they work, sometimes they don't. SMS is much more reliable. Let's go find another place to store our projects. Our projects are feeding their copilot IA and we have to pay for it. These people are scoundrels, when Github was sold to Microsoft people rumored that things will get bad here: they had reason.

Github interface is full of bugs, and features are flakey. C'mon people, begin by fixing bugs instead of trying to fix the world and lecture people what to do.
